conf/eslint.json

```diff
@@ -115,7 +115,7 @@
         "no-unneeded-ternary": "off",
         "no-unreachable": "error",
         "no-unsafe-finally": "error",
-        "no-unsafe-negation": "off",
+        "no-unsafe-negation": "error",
```

Comment:
This is a good catch! Should these changes be a separate PR, though, just in case?